NitroxClient/Communication/Packets/Processors/BulkheadDoorStateChangedProcessor.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.MonoBehaviours.CinematicController;
+using NitroxClient.Unity.Helper;
+using UnityEngine;
+using UWE;
+
+public class BulkheadDoorStateChangedProcessor : ClientPacketProcessor<BulkheadDoorStateChanged>
+{
+
+    private readonly PlayerManager remotePlayerManager;
+
+    public BulkheadDoorStateChangedProcessor(PlayerManager remotePlayerManager)
+    {
+        this.remotePlayerManager = remotePlayerManager;
+    }
+
+
+    public override void Process(BulkheadDoorStateChanged packet)
+    {
+        if (!NitroxEntity.TryGetObjectFrom(packet.Id, out GameObject bulkheadDoor))
+        {
+            Log.Error($"[BulkheadDoorStateChangedProcessor] Couldn't find GameObject for {packet.Id}");
+            return;
+        }
+
+        if (!remotePlayerManager.TryFind(packet.PlayerId, out RemotePlayer otherPlayer))
+        {
+            Log.Error($"[BulkheadDoorStateChangedProcessor] Couldn't find {nameof(RemotePlayer)} for {packet.PlayerId}");
+            return;
+        }
+
+        if (!bulkheadDoor.TryGetComponentInChildren(out BulkheadDoor door))
+        {
+            Log.Info("[BulkheadDoorStateChangedProcessor] Unable to find BulkheadDoor");
+            return;
+        }
+
+        bool prevState = !packet.IsOpen;
+
+        door.animator.SetBool("player_in_front", packet.IsFacingDoor);
+
+        PlayerCinematicController cinematicController = GetPlayerCinematicController(door, prevState, packet.IsFacingDoor);
+        MultiplayerCinematicController multiPlayerCinematicController = MultiplayerCinematicController.Initialize(cinematicController);
+
+        multiPlayerCinematicController.CallStartCinematicMode(otherPlayer);
+        PlayDoorSound(door, packet.IsOpen);
+
+        CoroutineHost.StartCoroutine(SetDoorStateAfterDelay(door, packet.IsOpen, 3.0f));
+    }
+
+    private void PlayDoorSound(BulkheadDoor door, bool isOpen)
+    {
+        bool isFacingDoor = door.GetSide();
+        string soundPath = GetDoorSoundPath(isOpen, isFacingDoor);
+        FMODAsset bulkheadOpenFrontAsset = Resources.Load<FMODAsset>(soundPath);
```

Comment:
Manually loading the sound seem not ideal. Either you whitelist the sound and our sound-system does the rest or you play it in regards to volume calculation. Depending on the type of sound you can use a easy method like in `NitroxClient.Communication.Packets.Processors.ToggleLightsProcessor.cs` or complex like the "play" methods in `NitroxClient.MonoBehaviours.FMODEmitterController.cs`